stable/express-gateway/README.md

```diff
@@ -0,0 +1,122 @@
+## Express Gateway
+
+[Express Gateway](https://express-gateway.io) is an API Gateway that sits at the heart of any microservices architecture.
+## TL;DR;
+
+```bash
+$ helm install stable/express-gateway
+```
+
+## Introduction
+
+This chart bootstraps all the components needed to run Express Gateway on a [Kubernetes](http://kubernetes.io)
+cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.9+
+- PV provisioner support in the underlying infrastructure if persistence
+  is needed for Express Gateway datastore (backed by Redis)
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```bash
+$ helm install --name my-release stable/express-gateway
+```
+
+> **Tip**: List all releases using `helm list`
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```bash
+$ helm delete my-release
+```
+
+The command removes all the Kubernetes components associated with the
+chart and deletes the release.
+
+## Configuration
+
+### General Deployment Configuration Parameters
+
+The following table lists the configurable parameters of the Kong chart
```

Comment:
it mentions Kong - but should it be Express Gateway  - there seems to be more references to Kong as well

Reply:
Correct, I've been using their Readme as a baseline. I'll fix these right now.